src/modules/countNumOfComments.test.js

```diff
@@ -0,0 +1,36 @@
+// import { countNumOfComments } from './commentsItemDetail.js';
```

Comment:
Please consider removing unused code in the code, unused code makes your work look unprofessional